packages/wrangler/src/index.ts

```diff
@@ -625,6 +625,11 @@ export function createCLIParser(argv: string[]) {
 								type: "string",
 								demandOption: true,
 							})
+							.option("content", {
+								describe: "Show script content for given deployment ID",
+								type: "boolean",
+								default: false,
+							})
```

Comment:
Thank you!

Reply:
@Skye-31 thank you for bringing the issue up!